mappings/net/minecraft/world/ViewableWorld.mapping

```diff
@@ -10,6 +10,14 @@ CLASS bbt net/minecraft/world/ViewableWorld
 		ARG 3 minZ
 		ARG 4 maxX
 	METHOD a getClosestPlayer (Lagv;D)Larb;
+	METHOD a getCollisionVoxelShapes (Lagv;Lclz;DDD)Ljava/util/stream/Stream;
+		ARG 3 x
+		ARG 5 y
+		ARG 7 z
+	METHOD a getCollsionVoxelShapes (Lagv;Lclz;Ljava/util/Set;DDD)Ljava/util/stream/Stream;
```

Comment:
No it gets the VoxelShape objects within a surrounding area for you to then iterate through and check collisions

Reply:
The classes dont have names because ive named them in here but i use it this way
`class_3538<VoxelShape> voxels = new class_3538(entity.world.method_8609(entity, boundingBox, moveX, moveY, moveZ));`
`moveY = VoxelShapes.method_1085(Direction.Axis.Y, boundingBox, voxels.method_15418(), moveY);`

Its the line from the first snippet method_8609 And class_3538 is the suggested LoopingStream

Reply:
No, I mean, there's a typo

Reply:
OHHH i thought you ment you wanted it to be getCollision overall. I was thinking wait that aint right